src/Core/Content/Media/File/FileFetcher.php

```diff
@@ -142,15 +142,8 @@ private function getUrlFromRequest(Request $request): string
      */
     private function openSourceFromUrl(string $url)
     {
-        $streamContext = stream_context_create([
-            'http' => [
-                'follow_location' => 0,
-                'max_redirects' => 0,
-            ],
-        ]);
-
         try {
-            $inputStream = @fopen($url, 'rb', false, $streamContext);
+            $inputStream = @fopen($url, 'rb');
```

Comment:
Can you can think of a way to test this, so we can verify that we don't accidentally break that in the future again?

Reply:
I don't know, I tested it with our image provider (Plentymarkets).